Transfer/Tests/CreateTransferPresenterTests.swift

```diff
@@ -0,0 +1,113 @@
+#if !COCOAPODS
+import Common
+#endif
+import Hippolyte
+import HyperwalletSDK
+@testable import Transfer
+import XCTest
+
+class TransferTests: XCTestCase {
+    private var presenter: CreateTransferPresenter!
+    private var mockView = MockCreateTransferView()
+
+    private let clientTransferId = "client-trasnsfer-id-123456789"
+    private let sourceToken = "trm-123456789"
+
+    override func setUp() {
+        Hyperwallet.setup(HyperwalletTestHelper.authenticationProvider)
+    }
+
+    override func tearDown() {
+        if Hippolyte.shared.isStarted {
+            Hippolyte.shared.stop()
+        }
+        mockView.resetStates()
+    }
+
+    private func initializePresenterEmptySource() {
+        presenter = CreateTransferPresenter(clientTransferId, nil, view: mockView)
+        presenter.loadCreateTransfer()
+    }
+
+    private func initializePresenterWithSource() {
+        presenter = CreateTransferPresenter(clientTransferId, sourceToken, view: mockView)
+        presenter.loadCreateTransfer()
+    }
+
+    private func initializePresenterEmptyTransferMethods() {
+        let response = HyperwalletTestHelper.noContentHTTPResponse()
+        let url = String(format: "%@%@", HyperwalletTestHelper.userRestURL, "/transfer-methods?")
+        HyperwalletTestHelper.setUpMockServer(request:
+            HyperwalletTestHelper.buildGetRequestRegexMatcher(pattern: url, response))
+    }
+
+    func testInitializeSectionsWhenSelectedTransferMethodIsNil() {
```

Comment:
rename to testInitializeSections_selectedTransferMethodIsNil

Reply:
renamed